tests/v2/08_full_workflow/test_full_workflow_slurm_v2.py

```diff
@@ -44,6 +51,22 @@ async def test_full_workflow_slurm(
         client=client,
     )
 
+    # https://github.com/fractal-analytics-platform/fractal-server/issues/1460
+    artifacts = os.listdir(fractal_runner_working_base_dir)
+    for item in [
+        "0_par_0000000.log",
+        "0_par_0000001.log",
+        "1_par_0000000.log",
+        "1_par_0000001.log",
+        HISTORY_FILENAME,
+        FILTERS_FILENAME,
+        IMAGES_FILENAME,
+        WORKFLOW_LOG_FILENAME,
+    ]:
+        assert item in os.listdir(
```

Comment:
Is it possible to move this assertion block into the `full_workflow` function?
This way, we could simply use the job `working_dir`, rather than `artifacts[0]`.
Also: the assertion hold for both backends, even if for `local` they are a bit trivial.